### DISCUSSION TITLE
🚩This feature is very important for Shopify apps

### DISCUSSION THREAD

**Discussion by ecomdeveloperbd** [inspireui/support#5699] [New Feature/Ideas Requests]
🚩This feature is very important for Shopify apps

1. Currency conversion option - very important for dropshipping
2. Photo review option is more important - very important for dropshipping
3. Any marketing tools integration (SMS Marketing & Email) - -very important for dropshipping
4. from visitor traffic source from apps --- analysis
5.  AI Camera or Image search - Product search

**Comment by jason-rotpunkt** [comment]
Dear sender,

Thank you for your e-mail. I have left the company. For enquiries regarding your PV system, please send an e-mail to Mr. Ho at ***@***.***, or call our main office at 6394 9349.

Thank you.

Best Regards,
Jonathan TAN
